MobileCenter/MobileCenter/Internals/Sender/MSHttpSender.m

```diff
@@ -334,12 +327,20 @@ - (NSString *)hideSecret:(NSString *)secret {
 
   // Hide everything if secret is shorter than the max number of displayed characters.
   NSUInteger appSecretHiddenPartLength =
-      (secret.length > kMSMaxCharactersDisplayedForAppSecret ? secret.length - kMSMaxCharactersDisplayedForAppSecret
-                                                             : secret.length);
+          (secret.length > kMSMaxCharactersDisplayedForAppSecret ? secret.length - kMSMaxCharactersDisplayedForAppSecret
+                  : secret.length);
   NSString *appSecretHiddenPart =
-      [@"" stringByPaddingToLength:appSecretHiddenPartLength withString:kMSHidingStringForAppSecret startingAtIndex:0];
+          [@"" stringByPaddingToLength:appSecretHiddenPartLength withString:kMSHidingStringForAppSecret startingAtIndex:0];
   return [secret stringByReplacingCharactersInRange:NSMakeRange(0, appSecretHiddenPart.length)
                                          withString:appSecretHiddenPart];
 }
 
+- (NSArray *)retryIntervals {
```

Comment:
Please document that this is the "empty" implementation that is intended to be overwritten.